Assets/MRTK/Providers/OculusXRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Windows.Utilities;
+using Microsoft.MixedReality.Toolkit.XRSDK.Input;
+using System;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.WindowsMixedReality
+{
+    /// <summary>
+    /// Manages XR SDK devices on the Oculus platform.
+    /// </summary>
+    [MixedRealityDataProvider(
+        typeof(IMixedRealityInputSystem),
+        SupportedPlatforms.Android,
+        "XRSDK Oculus Quest Device Manager")]
```

Comment:
Do we want this to be valid for desktop Oculus too? Maybe a question directed at @wiwei too

```suggestion
        SupportedPlatforms.Android | SupportedPlatforms.WindowsStandalone,
        "XRSDK Oculus Device Manager")]
```